src/assets/wise5/components/common/cRater/crater-rubric/crater-rubric.component.html

```diff
@@ -0,0 +1,25 @@
+<div>
+  <h1>
+    Idea detection rubric
+    <button (click)="closeDialog()"><mat-icon>close</mat-icon></button>
+  </h1>
+  @if (data.cRaterRubric.description && data.cRaterRubric.description !== '') {
```

Comment:
No need to wrap in object
```suggestion
  @if (cRaterRubric.description && cRaterRubric.description !== '') {
```